draft-ietf-capport-architecture.xml

```diff
@@ -433,22 +415,13 @@
                will not need to wait for a network failure to refresh its login. On receipt of
                preemptive notification, the User Equipment can prompt the user to refresh.
              </t>
-             <t>The protocol SHOULD have a method of identifying the source of signals.</t>
-             <t>The signal SHOULD NOT include any information other than a prompt to contact the API,
-                and any information necessary for validation.</t>
-             <t>
-                There is no requirement that the protocol be reliable.
-                Thus, The protocol SHOULD continue to send signals as long as the state which
-                triggered the first signal holds, subject to reasonable rate limiting.
-                This is necessary to handle network loss or rate limits between the Enforcement
-                Device and the User Equipment.
-             </t>
+             <t>The signal SHOULD NOT include any information other than a prompt to contact
```

Comment:
Rather than being a prompt to contact the API, can we call this "an indication that traffic is restricted, which can be used as a prompt to re-contact the API"?